src/v0/destinations/google_adwords_offline_conversions/config.js

```diff
@@ -1,6 +1,6 @@
 const { getMappingConfig } = require('../../util');
 
-const API_VERSION = 'v16';
+const API_VERSION = 'v17';
```

Comment:
![image](https://github.com/user-attachments/assets/102f2f9f-bb10-46e6-9952-459db6e9d2b0)

Reply:
But it will sunset on June 4th. Will create another task to update to the latest version.